minimal.ini

```diff
@@ -0,0 +1,70 @@
+###
```

Comment:
Should probably be renamed container.ini, maybe make use of an already existing .ini-file.